lib/components/screen_switch.dart

```diff
@@ -32,7 +32,7 @@ class _ScreenSwitchState extends State<ScreenSwitch> {
           child: Column(
             children: [
               Container(
-                padding: const EdgeInsets.all(24.0),
+                padding: const EdgeInsets.all(16.0),
```

Comment:
ok bro what the flip
![image](https://github.com/Bartosiki/DW-Flutter-App/assets/84678175/aa0aa407-9a6c-4bef-879d-6e00f2ee7d64)

Reply:
What da shiet 
![9695e7845d224d235238f0bf67ef79e4](https://github.com/Bartosiki/DW-Flutter-App/assets/97341476/9c86dd4a-82c0-40cc-812b-70eb5c7b9442)